### DISCUSSION TITLE
introductory proposal

### DISCUSSION THREAD

**Discussion by hbozkurtt** [IBAX-io/go-ibax#73] [Rewards]
a limited number of awards can be distributed and presented to participants through educational and promotional events to be held in countries

for example, a live broadcast promotional event on youtube, followed by a gift of 1 ibxc coin to participants

**Comment by kelessburakk** [comment]
Anlamadım ne demek istediniz

20 Oca 2022 Per 22:22 tarihinde hbozkurtt ***@***.***> şunu
yazdı:
